app/src/main/java/ch/epfl/polychef/fragments/PostRecipeFragment.java

```diff
@@ -488,6 +499,57 @@ private void getAndCheckIngredients() {
         wrongInputs.put("Ingredients", true);
     }
 
+
+    private void initializeFromOriginalRecipe(Recipe originalRecipe) {
+        postingAModifiedRecipe=true;
+        this.originalRecipe=originalRecipe;
+
+        hideImageComponents();
+
+        EditText prepTimeInput = getView().findViewById(R.id.prepTimeInput);
+        prepTimeInput.setText(Integer.toString(originalRecipe.getEstimatedPreparationTime()));
```

Comment:
Quickfix the 4 Cirrus warnings ? (add String.format)

Reply:
I will solve this